code/modules/mob/living/carbon/life.dm

```diff
@@ -256,9 +256,11 @@
 				continue
 			if(istype(M, /mob/living/carbon) && stat != 2)
 				if(M.stat == 2)
-					M.death(1)
-					stomach_contents.Remove(M)
-					qdel(M)
+					if(!istype(src, /mob/living/carbon/alien))

```

Comment:
snowflakey, add a ```digestsStomachContents``` variable to carbon or living. and set it on Aliens so it's 0.